integrationTests/chainSimulator/bridge/simpleEsdt_test.go

```diff
@@ -0,0 +1,446 @@
+			cfg.EpochConfig.EnableEpochs.DynamicESDTEnableEpoch = 0
+		},
+	})
+	require.Nil(t, err)
+	require.NotNil(t, cs)
+
+	defer cs.Close()
+
+	err = cs.GenerateBlocksUntilEpochIsReached(4)
+	require.Nil(t, err)
+
+	nodeHandler := cs.GetNodeHandler(0)
+	systemContractDeploy := chainSim.GetSysContactDeployAddressBytes(t, nodeHandler)
+
+	initialAddress := "erd1l6xt0rqlyzw56a3k8xwwshq2dcjwy3q9cppucvqsmdyw8r98dz3sae0kxl"
+	initialAddrBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(initialAddress)
+	err = cs.SetStateMultiple([]*dtos.AddressState{
+		{
+			Address: initialAddress,
+			Balance: "10000000000000000000000",
+		},
+		{
+			Address: esdtSystemAccount, // init sys account
+		},
+	})
+	require.Nil(t, err)
+	nonce := uint64(0)
+
+	err = cs.GenerateBlocks(1)
+	require.Nil(t, err)
+
+	contractAddress := chainSim.DeployContract(t, cs, initialAddrBytes, &nonce, systemContractDeploy, "", simpleEsdtTestWasmPath)
+
+	tokens := createAllEsdtTypes(t, cs, initialAddrBytes, contractAddress, &nonce)
+	for _, token := range tokens {
+		checkMetaDataInAccounts(t, cs, token, initialAddress, token.Amount)
+	}
+
+	burnAllEsdtTypes(t, cs, initialAddrBytes, contractAddress, &nonce, tokens)
+	for _, token := range tokens {
+		checkMetaDataInAccounts(t, cs, token, initialAddress, big.NewInt(0))
+	}
+}
+
+func createAllEsdtTypes(
+	t *testing.T,
+	cs chainSim.ChainSimulator,
+	sender, contract []byte,
+	nonce *uint64,
+) []chainSim.ArgsDepositToken {
+	tokens := make([]chainSim.ArgsDepositToken, 0)
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab1-TKN-fasd35",
+		Nonce:      uint64(0),
+		Amount:     big.NewInt(14556666767),
+		Type:       core.Fungible,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab1-NFTV2-1ds234",
+		Nonce:      uint64(3),
+		Amount:     big.NewInt(1),
+		Type:       core.NonFungibleV2,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab2-DNFT-fdfe3r",
+		Nonce:      uint64(22),
+		Amount:     big.NewInt(1),
+		Type:       core.DynamicNFT,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab2-SFT-gw4fw2",
+		Nonce:      uint64(345),
+		Amount:     big.NewInt(1421),
+		Type:       core.SemiFungible,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab4-DSFT-g43g2s",
+		Nonce:      uint64(88),
+		Amount:     big.NewInt(1534),
+		Type:       core.DynamicSFT,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab5-META-1ds234",
+		Nonce:      uint64(44),
+		Amount:     big.NewInt(6231),
+		Type:       core.MetaFungible,
+	})
+	tokens = append(tokens, chainSim.ArgsDepositToken{
+		Identifier: "ab5-DMETA-f23g2f",
+		Nonce:      uint64(721),
+		Amount:     big.NewInt(162367),
+		Type:       core.DynamicMeta,
+	})
+
+	var txData string
+	for _, token := range tokens {
+		if token.Type == core.Fungible {
+			txData = createLocalMintArgs(token.Identifier, token.Amount)
```

Comment:
Can you move `createLocalMintArgs` + `createNftArgs` right below here or somewhere where they are spatially located on top of each other?

Reply:
sure